doc/release-process.md

```diff
@@ -312,3 +312,22 @@ bitcoin.org (see below for bitcoin.org update instructions).
   - Create a [new GitHub release](https://github.com/bitcoin/bitcoin/releases/new) with a link to the archived release notes.
 
   - Celebrate
+
+### Additional information
+
+#### How to calculate `m_assumed_blockchain_size` and `m_assumed_chain_state_size`
+
+Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
```

Comment:
You could mention up here that the node needs to be synced, i.e `All values should be taken from a fully synced node.`, and save repeating it 4 times below.

You could also do the same for adding overhead. i.e `Add 5-10% overhead to all values.` and save repeating, plus any ambiguity.

Reply:
That makes sense. Thanks!